### ISSUE TITLE
WCAG contrast breach on default buttons for image component

### ISSUE THREAD

**Issue by AndreasJacobsen** [sanity-io/sanity#2201]
**Describe the bug**

Default buttons in type image fails WCAG minimum contrast requirements. 

The "Upload" and "Select" buttons have a contrast ratio of 3.41 while the minimum WCAG AA requirement is 4.5. 

See [WCAG 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) for more info

**To Reproduce**
Create an image component, this one should do the trick
```
{
      title: "Contrast breach",
      name: "someImageComponent",
      type: "image",
}
```

Go the the component and look at the "Upload" and "Select" buttons to see the contrast breach. 

**Expected behavior**

The text should have a contrast ratio of at least 4.5.

**Screenshots**
![Screenshot of image component with two buttons](https://user-images.githubusercontent.com/8527173/103480175-1757cb80-4dd3-11eb-8af3-a70fed0807f9.png)

![Screenshot of chrome dev tools showing color picker with contrast checker](https://user-images.githubusercontent.com/8527173/103480240-6ef63700-4dd3-11eb-9385-5377fe7d6412.png)

**Which versions of Sanity are you using?**

```
@sanity/cli                 2.0.1 (latest: 2.1.0)
@sanity/base                2.1.3 (up to date)
@sanity/cli                 2.1.0 (up to date)
@sanity/code-input          2.1.0 (up to date)
@sanity/color-input         2.1.0 (up to date)
@sanity/components          2.1.0 (up to date)
@sanity/core                2.1.2 (up to date)
@sanity/dashboard           2.1.0 (up to date)
@sanity/default-layout      2.1.3 (up to date)
@sanity/default-login       2.1.0 (up to date)
@sanity/desk-tool           2.1.3 (up to date)
@sanity/production-preview  2.1.0d (up to date)
@sanity/vision              2.1.0 (up to dates)
```

**What operating system are you using?**
MacOS Catelina

**Which versions of Node.js / npm are you running?**

```
6.13.4
v12.16.1
```

**Comment by rexxars** [comment]
Thanks for reporting! The image and file inputs have received some major brushups to be both more compact and more in line with the rest of the studio!
